.github/workflows/review_cypress_e2e_tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Cypress end-to-end tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - backend/**
+      - client/**
```

Comment:
would the idea to be running the tests on each PR? for that seems it needs

```
on:
  pull_request:
```

Reply:
@stavares843 you mean:
on:
  pull_request:
    types: [opened]

Reply:
i used 

on:
pull_request:

and worked fine

example 

https://github.com/FightPandemics/FightPandemics-android/pull/38/files

Reply:
@stavares843 

ok, so I will ask about it